src/libsyntax/parse/parser.rs

```diff
@@ -1802,6 +1802,20 @@ impl<'a> Parser<'a> {
             // First, parse an identifier.
             let identifier = self.parse_path_segment_ident()?;
 
+            if
+                self.check(&token::ModSep) &&
```

Comment:
stylistically I don't think we put line breaks between the `if` and the condition expression in cases like this.